packages/docs-app/src/examples/timezone-examples/timezonePickerExample.tsx

```diff
@@ -20,25 +22,28 @@ export interface ITimezonePickerExampleState {
 export class TimezonePickerExample extends React.PureComponent<IExampleProps, ITimezonePickerExampleState> {
     public state: ITimezonePickerExampleState = {
         disabled: false,
+        showCustomTarget: false,
         showLocalTimezone: true,
         targetDisplayFormat: TimezoneDisplayFormat.COMPOSITE,
         timezone: "",
     };
 
     private handleDisabledChange = handleBooleanChange(disabled => this.setState({ disabled }));
     private handleShowLocalChange = handleBooleanChange(showLocalTimezone => this.setState({ showLocalTimezone }));
+    private handleCustomChildChange = handleBooleanChange(showCustomTarget => this.setState({ showCustomTarget }));
     private handleFormatChange = handleStringChange((targetDisplayFormat: TimezoneDisplayFormat) =>
         this.setState({ targetDisplayFormat }),
     );
 
     public render() {
-        const { timezone, targetDisplayFormat, disabled, showLocalTimezone } = this.state;
+        const { timezone, targetDisplayFormat, disabled, showCustomTarget, showLocalTimezone } = this.state;
 
         const options = (
             <>
                 <H5>Props</H5>
                 <Switch checked={showLocalTimezone} label="Show local timezone" onChange={this.handleShowLocalChange} />
                 <Switch checked={disabled} label="Disabled" onChange={this.handleDisabledChange} />
+                <Switch checked={showCustomTarget} label="Custom target" onChange={this.handleCustomChildChange} />
```

Comment:
can we put this in an `<H5>Example</H5>` section at the end to clarify that it's more involved than a single prop change?

see https://blueprintjs.com/docs/#core/components/callout for inspo.